.github/workflows/test_with_tox.yml

```
@@ -84,9 +84,11 @@ jobs:
             id: py
             run: |
                 versions='py=["3.14", "3.13", "3.12", "3.11", "3.10"]'
-                if [ "$GITHUB_EVENT_NAME" == "pull_request" ]; then
-                    versions='py=["3.14", "3.10"]'
-                fi
+
+                # if [ "$GITHUB_EVENT_NAME" == "pull_request" ]; then
+                #     versions='py=["3.14", "3.10"]'
+                # fi
```

```suggestion
                if [ "$GITHUB_EVENT_NAME" == "pull_request" ]; then
                    versions='py=["3.14", "3.10"]'
                fi
```

revert before merging: need to see if some of the errors are python version specific